leakcanary-analyzer/src/main/java/leakcanary/LeakReference.kt

```diff
@@ -27,8 +27,7 @@ import java.io.Serializable
  */
 data class LeakReference(
   val type: Type,
-  val name: String,
-  val value: String
```

Comment:
`value` was unused (toString wasn't used anymore), so this saves a whole bunch of string builder calls when enqueuing